examples/AN00124_CDC_VCOM_class/doc/rst/AN00124.rst

```diff
@@ -1,3 +1,5 @@
+.. version:: 2.0.2
```

Comment:
Because we will eventually (I hope) automate updates to the version number, I would like to see this information go into a separate file in the doc/rst directory which an `.. include::` directive pulls into the App Note rst file.